loki/README.md

```diff
@@ -36,27 +35,43 @@ To get all the customization options, follow this instructions to set up a paral
     helm repo update
     ```
 
-### Install the Loki stack
+## Install Loki
+
+>**NOTE** Loki can be installed in different [Deployment modes](https://grafana.com/docs/loki/latest/fundamentals/architecture/deployment-modes/) dependent on your scalability needs and storage requirements. This instructions Loki in a lightweight in-cluster solution that does not fulfil production-grade qualities. Consider using a scalable setup based on an object storage backend instead (see [Simple scalable deployment of Grafana Loki with Helm](https://grafana.com/docs/loki/latest/installation/helm/install-scalable/)).
+
+### Installation
 
 You install the Loki stack with a Helm upgrade command, which installs the chart if not present yet.
 
-You can either choose an installation based on [Promtail](https://grafana.com/docs/loki/latest/clients/promtail/), which is the log collector recommended by Loki and provides a ready-to-use setup.
-Alternatively, you can use Kyma's LogPipeline feature based on Fluent Bit.
+```bash
+helm upgrade --install --create-namespace -n ${KYMA_NS} ${HELM_LOKI_RELEASE} grafana/loki -f https://raw.githubusercontent.com/kyma-project/examples/main/loki/loki-values.yaml
+```
+
+In any case, you can either use the [loki-values.yaml](./loki-values.yaml) provided in this `loki` folder, which contains customized settings deviating from the default settings, or create your own `values.yaml` file. The prepared `values.yaml` file activates the `singleBinary` mode and disables additional components which are usually used when running Loki as a central backend. 
```

Comment:
The thought jumps from "provided file" to "own file" back to "provided file". Maybe we can simplify this a bit:

```suggestion
In any case, you can either create your own `values.yaml` file, or use the [loki-values.yaml](./loki-values.yaml) provided in this `loki` folder, which contains customized settings deviating from the default settings: The provided `values.yaml` file activates the `singleBinary` mode and disables additional components that are usually used when running Loki as a central backend. 
```